paraatm/tests/test_para_atm.py

```diff
@@ -2,20 +2,21 @@
 import pandas as pd
 import numpy as np
 import os
-
+import torch
 from paraatm.io.nats import read_nats_output_file, NatsEnvironment
 from paraatm.io.gnats import read_gnats_output_file, GnatsEnvironment
 from paraatm.io.iff import read_iff_file
 from paraatm.io.utils import read_csv_file
 from paraatm.safety.ground_ssd import ground_ssd_safety_analysis
 from paraatm.rsm.gp import SklearnGPRegressor
 from paraatm.simulation_method.vcas import VCAS
+from paraatm.simulation_method.aviationr import AviationRisk
 
 from . import nats_gate_to_gate
 from . import gnats_gate_to_gate
 
 # Change this to False to test NATS instead of GNATS
-USE_GNATS = True
+USE_GNATS = False
```

Comment:
You may have changed this value during testing.  Please change it back to `True`.

Reply:
Got it. Thanks!